development/build/scripts.js

```diff
@@ -471,6 +471,33 @@ function createScriptTasks({
         version,
         applyLavaMoat,
       }),
+      () => {
+        // MV3 injects inpage into the tab's main world, but in MV2 we need
+        // to do it manually:
+        if (process.env.ENABLE_MV3) {
```

Comment:
we may be doing it elsewhere but as a point of style I like to read the ENV at the beginning of the module, and use that state elsewhere

Reply:
I'd like to keep drive-by refactorings out of this PR, if possible -- as it's already hard enough to keep reviews on this one 😅  . I've created an issue for your refactoring request: https://github.com/MetaMask/metamask-extension/issues/22831.